MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -99,15 +103,17 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
     }
     
     private func showNextQuestionOrResults() {
+        
         if currentQuestionIndex == questionsAmount - 1 {
             var text = "Ваш результат: \(correctAnswers)/\(questionsAmount)"
             if let statisticService = statisticService {
                 statisticService.store(correct: correctAnswers, total: questionsAmount)
                 let bestGame = statisticService.bestGame
-                text += "\nКоличество сыгранных квизов: \(statisticService.gamesCount)"
-                text += "\nРекорд: \(bestGame.correct)/\(bestGame.total) (\(bestGame.date.dateTimeString))"
-                text += "\nСредняя точность: \(String(format: "%.2f", statisticService.totalAccuracy))%"
-                
+                text += """
+                \nКоличество сыгранных квизов: \(statisticService.gamesCount)
+                Рекорд: \(bestGame.correct)/\(bestGame.total) (\(bestGame.date.dateTimeString))
+                Средняя точность: \(String(format: "%.2f", statisticService.totalAccuracy))%
+                """
```

Comment:
если используешь многострочные строки, то \n больше не нужен.  Перевод строки будет эквивалентен новой строке в коде
```suggestion
                Количество сыгранных квизов: \(statisticService.gamesCount)
                Рекорд: \(bestGame.correct)/\(bestGame.total) (\(bestGame.date.dateTimeString))
                Средняя точность: \(String(format: "%.2f", statisticService.totalAccuracy))%
                """
```